### PR TITLE
added spring

### PR DESCRIPTION
**Pull request by MaVaYuran**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -8,15 +8,34 @@
...
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-context</artifactId>
+            <version>${spring.version}</version>
+        </dependency>
+<!--        <dependency>-->
+<!--            <groupId>org.projectlombok</groupId>-->
+<!--            <artifactId>lombok</artifactId>-->
+<!--            <version>1.18.34</version>-->
+<!--        </dependency>-->
```
Comment (Alexmansar) [inline pom.xml:48]: remove it if you don't use it)

File: pom.xml [hunk 2]
```diff
@@ -8,15 +8,34 @@
     <artifactId>andersen</artifactId>
     <version>1.0-SNAPSHOT</version>
     <packaging>jar</packaging>
+
     <properties>
         <maven.compiler.source>21</maven.compiler.source>
         <maven.compiler.target>21</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+        <spring.version>6.1.12</spring.version>
```
Comment (Alexmansar) [inline pom.xml:17]: 👍

File: pom.xml [hunk 3]
```diff
@@ -63,7 +82,7 @@
                             <archive>
                                 <manifest>
                                     <mainClass>
-                                        by.mariayuran.homework.TicketService
+                                        by.mariayuran.by.mariayuran.homework.TicketService
```
Comment (Alexmansar) [inline pom.xml]: its correct?

File: src/main/resources/hibernate.cfg.xml [hunk 4]
```diff
@@ -12,17 +12,17 @@
         <property name="connection.password">root</property>
 
         <!-- SQL dialect -->
-        <property name="dialect">org.hibernate.dialect.PostgreSQLDialect</property>
+        <property name="dialect">org.by.mariayuran.hibernate.dialect.PostgreSQLDialect</property>
```
Comment (Alexmansar) [inline src/main/resources/hibernate.cfg.xml]: and here))

File: src/main/resources/hibernate.cfg.xml [hunk 5]
```diff
@@ -12,17 +12,17 @@
...
 
         <!-- Display SQL in console -->
         <property name="show_sql">true</property>
 
         <!-- Create/update tables automatically -->
-        <property name="hibernate.hbm2ddl.auto">update</property>
+        <property name="by.mariayuran.hibernate.hbm2ddl.auto">update</property>
```
Comment (Alexmansar) [inline src/main/resources/hibernate.cfg.xml]: and here

File: src/main/java/by/mariayuran/Runner.java [hunk 6]
```diff
@@ -0,0 +1,9 @@
+package by.mariayuran;
+
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+
+public class Runner {
+    public static void main(String[] args) {
+        AnnotationConfigApplicationContext context=new AnnotationConfigApplicationContext(SpringConfig.class);
```
Comment (Alexmansar) [inline src/main/java/by/mariayuran/Runner.java:9]: now, use this context, find your beans and use some methods from these beans

**Review by Alexmansar — CHANGES_REQUESTED** [review]
@MaVaYuran Hi) please, check and fix  comments

File: pom.xml (hunk 3 above)
Comment (MaVaYuran) [reply pom.xml]: no, how is it?

Comment (Alexmansar) [reply pom.xml]: So just fix it. The IDE may have done something wrong)

File: src/main/java/by/mariayuran/SpringConfig.java [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+    @Bean
+    public LocalSessionFactoryBean sessionFactory(DataSource dataSource) {
+        LocalSessionFactoryBean sessionFactory = new LocalSessionFactoryBean();
+        sessionFactory.setHibernateProperties(hibernateProperties());
+       sessionFactory.setDataSource(dataSource);
+        sessionFactory.setAnnotatedClasses(
+                User.class,
+                Ticket.class
+        );
+        System.out.println(sessionFactory);
```
Comment (Alexmansar) [inline src/main/java/by/mariayuran/SpringConfig.java]: or remove, or use logging)

File: src/main/java/by/mariayuran/SpringConfig.java [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+        dataSource.setPassword(password);
+        return dataSource;
+    }
+
+    @Bean
+    public LocalSessionFactoryBean sessionFactory(DataSource dataSource) {
+        LocalSessionFactoryBean sessionFactory = new LocalSessionFactoryBean();
+        sessionFactory.setHibernateProperties(hibernateProperties());
+       sessionFactory.setDataSource(dataSource);
+        sessionFactory.setAnnotatedClasses(
```
Comment (Alexmansar) [inline src/main/java/by/mariayuran/SpringConfig.java]: fix style

File: src/main/java/by/mariayuran/hibernate/HibernateRunner.java [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+//        System.out.println(userDAO.save(new User("Rafael", LocalDate.now())));
+//        System.out.println(userDAO.getUser(1));
+//        System.out.println(userDAO.delete(10));
+//
+//        TicketDAOImpl ticketDAO = new TicketDAOImpl(HibernateUtil.buildSessionFactory());
+//        User user = userDAO.getUser(3);
+//        System.out.println(ticketDAO.save(new Ticket(user,TicketType.DAY, LocalDate.now())));
+//        System.out.println(ticketDAO.getTicket(7));
+//        System.out.println(ticketDAO.getUserTickets(4));
+
```
Comment (Alexmansar) [inline src/main/java/by/mariayuran/hibernate/HibernateRunner.java]: remove or uncommented

**Review by Alexmansar — CHANGES_REQUESTED** [review]
@MaVaYuran Please, fix new and old comments)

**Pull request merged**